onnx/defs/math/defs.cc

```diff
@@ -424,6 +424,37 @@ max(0,x) + min(0,alpha*(exp(x/alpha)-1))
 
 static float celu_default_alpha = 1.0;
 
+TensorProto ToDimensionOneFloatTensor(float value) {
+  auto t = ToTensor(std::vector<float>({value}));
+  t.add_dims(1);
+  return t;
+}
+
+bool BuildContextDependentFunctionBodyCelu(
+    const FunctionBodyBuildContext& ctx,
+    const OpSchema& schema,
+    FunctionProto& functionProto) {
+  std::vector<FunctionBodyHelper::NodeDef> body;
+float alpha = ctx.getAttribute("alpha")->f();
```

Comment:
Use
```
   ctx.getAttribute("alpha") ? ctx.getAttribute("alpha")->f() : 1.0f
```

Reply:
Thanks for the reminder I was planning to checking for missing value but doesn't; this attribute already have a default value set in the schema? Also I think we should use the default const below. 

ctx.getAttribute("alpha") ? ctx.getAttribute("alpha")->f() : celu_default_alpha


---
In reply to: [410777048](https://github.com/onnx/onnx/pull/2725#discussion_r410777048) [](ancestors = 410777048)

Reply:
I think it is just safer. I am not sure if the check is needed (this can be called from ORT, but it can also be called from other ONNX client-code ... I recall that ORT fills in default values, but not sure if that also happens in the ONNX context).

Reply:
Safer for sure.

---
In reply to: [410779075](https://github.com/onnx/onnx/pull/2725#discussion_r410779075) [](ancestors = 410779075)